PSReadLine/KeyBindings.vi.cs

```diff
@@ -231,6 +231,7 @@ private void SetDefaultViBindings()
                 { Keys.Space,           MakeKeyHandler( DeleteChar,                   "DeleteChar") },
                 { Keys._0,              MakeKeyHandler( BackwardDeleteLine,           "BackwardDeleteLine") },
                 { Keys.Uphat,           MakeKeyHandler( DeleteLineToFirstChar,        "DeleteLineToFirstChar") },
+                { Keys.Underbar,        MakeKeyHandler( DeleteLine,                   "DeleteLine") },
```

Comment:
Can you please move this one right after the `Keys.D` (the other "DeleteLine")?

Also, this requires an update to the doc at https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/7.1/PSReadLine/About/about_PSReadLine.md#deleteline
/cc @sdwheeler

Reply:
[Added documentation](https://github.com/MicrosoftDocs/PowerShell-Docs/pull/6338) for the new <kbd>d</kbd><kbd>_</kbd> chord for the `DeleteLine` command.